### ISSUE TITLE
Accessibility, language learning, anonymous letters ideas

### ISSUE THREAD

**Issue by jaerlcruz** [hanshim5/Hackathon-2024#1]
Map to show accessibility support:
- Show areas that have high accessibility support, i.e. most establishments and buildings in an area can accommodate those in wheelchairs (ramps, elevators)
- Appeal: I have relatives that always want to travel to places like Japan, but Japan travel requires lots of walking, which they may be unable to do, and I'd like to pinpoint where in Japan I could take them (or anywhere else)
- challenges: highlighting density of accessible areas
- google maps API has an accessibilityoptions property for places we can use

Language learning, "word/phrase of the day":
- a fun app for aspiring polyglots, they can click different countries and get a random vocab word (or phrase!) for each one based on the main language of the country/region
- may help people find out what language they want to learn next by getting a preview of one?
- challenges: we would have to come up or use an AI maybe to come up with words or phrases to load. i dont know if there's a proper dataset available to implement this easily in 24hrs.
- google translate api to grab words (?)

Anonymous message map:
- inspired by the unsent project, this website will self destruct, and a little app on my PC called "desktop meadow," where each hour I receive an anonymous kind letter and have the option to send one out. users could choose a specific region to send to(?)
- basically just a neat simple way to spread kindness :]
- challenges: possibly requires a database since we're taking in more user input. i have worked with supabase before for a quick project like this but other options include mongodb and firebase. supabase is free for our purposes and beginner friendly from my experience.
